R/lockbox.R

```diff
@@ -63,7 +64,7 @@ lockbox.list <- function(lock, env) {
     align(lock[mismatches])
 
     ## And re-build our search path. Do so in the reverse order of dependencies.
-    rebuild(rev(lock))
+    rebuild(c(lock[vapply(lock, `[[`, logical(1), "is_dependency_package")], original_lock))
```

Comment:
Can you update the comment as well?

Doesn't this reload some packages twice? Or it only appears that way since loading is idempotent? We could do a `unique` if that's the case.